### PR TITLE
Write cert in Java keystore format

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -245,6 +245,8 @@ func main() {
 
 	log.Printf("wrote %s", certFile)
 
+	writeKeystore(certDir, key, certificate)
```
Comment (raghu67) [inline main.go:248]: Is it OK to assume Java KeyStore and the PEM Cert go to the same directory? Is that how we configure our JRE for Platform servers?

File: keystore.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	"crypto/x509"
+	"encoding/pem"
+	"github.com/lwithers/minijks/jks"
+	"io/ioutil"
+	"log"
+	"path"
+	"time"
+)
+
+func writeKeystore(certDir string, privkey *rsa.PrivateKey, pemCertificate []byte) {
```
Comment (raghu67) [inline keystore.go:14]: Is there anything we could do to Validate/Unit Test the writing of the KeyStore? I understand from the minijks that the format is actually not documented

**Review by raghu67 — APPROVED** [review]
Comments Inline

File: main.go (hunk 1 above)
Comment (johngmyers) [reply main.go:248]: Platform defaults to reading from etc under the install directory ("/service" for Docker). That won't work here, as we need it to go into a volume. So we'll probably have to configure the Platform service to read from wherever certificate-init-container puts the keystore. We could do that configuration change in the application code.

File: keystore.go (hunk 2 above)
Comment (johngmyers) [reply keystore.go:14]: I see two options:
1) Use minijks to read the keystore. That won't catch any consistent error in minijks itself.
2) Add a testing dependency on keytool. That seems a bit too heavyweight.

**Pull request merged**